### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by franceshg**
Tailwind and Flowbite Custom Theme styling

### CODE REVIEW

File: public/pink_horizontal_transparent.png
Comment (erinrolson) [inline public/pink_horizontal_transparent.png:1]: If we aren't using this version of the logo in the dashboard, I think we should remove it from the repo.

File: src/components/LogsTable.tsx [hunk 1]
```diff
@@ -394,80 +410,93 @@ function LogsTable() {
...
+            </Table.Head>
+            <Table.Body className="divide-y">
+              {tableData.length > 0 ? (
+                tableData.map((log) => {
+                  return (
+                    <Table.Row
+                      onClick={() => handleOpen(log)}
+                      key={log.log_id}
+                      className="hover:bg-[#6B778D] cursor-pointer"
+                      onClick={() => handleOpen(log)}
```
Comment (erinrolson) [inline src/components/LogsTable.tsx:444]: Repeated reference to the `onClick` handler needs to be removed.

File: src/components/LogsTable.tsx [hunk 2]
```diff
@@ -394,80 +410,93 @@ function LogsTable() {
...
+                  );
+                })
+              ) : (
+                <Table.Row>
+                  <Table.Cell colSpan={5} className="text-center p-4">
+                    No notification logs to display.
+                  </Table.Cell>
+                  <Table.Cell onClick={() => handleOpen(log)}>
+                    <p className="font-medium hover:underline">Log Details</p>
+                  </Table.Cell>
```
Comment (erinrolson) [inline src/components/LogsTable.tsx:476]: I don't believe this code belongs here and can be deleted.

**Comment by kwjolee** [comment]
All features are looking good.

**Pull request merged**